scripts/open_covid_data/open_covid_19_data.mcf

```diff
@@ -0,0 +1,972 @@
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalTest_COVID_19_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalTest_COVID_19
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID_19_Tested_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID_19_Tested
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_MedicalTest_COVID_19_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_MedicalTest_COVID_19
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalConditionIncident_COVID_19_PatientHospitalized
+typeOf: dcs:StatisticalVariable
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+medicalStatus: dcs:PatientHospitalized
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+
+Node: dcid:PolicyExtent_Legislation_COVID19Pandemic_GovernmentOrganization_SchoolClosure_SelectedAdministrativeAreas
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+policyType: dcs:SchoolClosure
+measuredProperty: dcs:policyExtent
+spatialCoverage: dcs:SelectedAdministrativeAreas
+
+Node: dcid:PolicyExtent_Legislation_COVID19Pandemic_GovernmentOrganization_SchoolClosure_AllAdministrativeAreas
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+policyType: dcs:SchoolClosure
+measuredProperty: dcs:policyExtent
+spatialCoverage: dcs:AllAdministrativeAreas
+
+Node: dcid:PolicyExtent_Legislation_COVID19Pandemic_GovernmentOrganization_SchoolClosure_Unknown
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+policyType: dcs:SchoolClosure
+measuredProperty: dcs:policyExtent
+spatialCoverage: dcs:Unknown
```

Comment:
Now realizing this causes the StatVar ID to be confusion. Let's change this enum to SpatialCoverageUnknown